include/HamiltonianBuilder/DOCI.hpp

```diff
@@ -0,0 +1,60 @@
+#ifndef GQCG_DOCI_HPP
+#define GQCG_DOCI_HPP
+
+
+#include "RestrictedHamiltonianBuilder.hpp"
+#include "FockSpace/FockSpace.hpp"
+
+#include <memory>
+
+
+
+namespace GQCG {
+
+
+/**
+ *  Doubly occupied configuration interaction builds a hamiltonian matrix
+ *  based on a wavefunction only containing doubly occupied configurations
+ *  This requires the alpha and beta Fock space to be identical
+ *  combinations of identical alpha, beta ONVs are used.
```

Comment:
I feel like this sentence is unnecessary

Reply:
Maybe it is poorly worded, but with two identical Fock spaces you can do FCI when you let the alpha and beta ONVs vary independently.

Reply:
indeed, word this differently